services/order.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict[str, int]],
+    username: str,
+    date: datetime = None
```

Comment:
The `date` parameter should specify whether it expects a `datetime.date` or `datetime.datetime` object to avoid confusion and ensure proper handling.